### ISSUE TITLE
Wrong modeling rules for a node variable of an object type instance

### ISSUE THREAD

**Issue by FKurniawanHuisman** [atSCM/atscm#354]
1. There is an object type called _MyObjectType1_ and it has a bool variable with modeling rule set to _New_.
Note: inside the square bracket is the type.

- _MyObjectType1_
  - _myVariable_ [_Boolean_] -> _New_

2. There is another object type that has an instance of _MyObjectType1_ inside it and the _myVariable_ of the instance is set to modeling rule Shared.
- _MyObjectType2_
  - _myObject_ [_MyObjectType1_]
    - _myVariable_ [_Boolean_] -> **_Shared_**

![image](https://user-images.githubusercontent.com/54705186/73663349-ae854200-469d-11ea-9fcb-d78b95ada4e7.png)

I attached the nodes.db of this nodes setup.
[nodes.zip](https://github.com/atSCM/atscm/files/4148500/nodes.zip)

3. Pulling the sources of these object types generates sources with correct modeling rules.
- _myVariable_ of _MyObjectType1_ object type has the modeling rule _New_.
- _myVariable_ of _myObject_ instance inside _MyObjectType2_ has modeling rule _Shared_.

4. The issue happens when pushing the sources to an **empty** nodes.db which doesn't have the object types.
The issue is _myVariable_ of _myObject_ instance inside _MyObjectType2_ is created with modeling rule _New_ instead of _Shared_.

<details><summary><strong>Environment</strong></summary>

| package       |               version |
| :------------ | --------------------: |
| OS            | Windows |
| atscm         |     1.0.0-beta.46|
| atscm-cli     |     0.9.0 |
| atvise server | 3.4.0|
| node          |          12.14.1 |
| npm           |           6.13.4 |
 
</details>

<details><summary><strong>Project configuration</strong></summary>

```javascript
// Atviseproject.babel.js

import { Atviseproject, NodeId } from 'atscm';

/**
 * atvise-scm configuration of framework.
 */
export default class Framework extends Atviseproject {

  /**
   * The atvise-server's host
   * @type {String}
   */
  static get host() {
    return 'localhost';
  }

  /**
   * The atvise-server ports to use.
   * @type {Object}
   * @property {Number} opc The OPC-UA port the atvise-server runs on.
   * @property {Number} http The HTTP port the atvise-server can be reached at.
   */
  static get port() {
    return {
      opc: 4840,
      http: 80,
    };
  }
  
  static get ignoreNodes() {
	return super.ignoreNodes.concat([
		new NodeId('SYSTEM.LIBRARY.ATVISE'),
		new NodeId('SYSTEM.INFORMATION'),
	]);
  }

  /**
   * The login to use. Return `false` if no login is required.
   * @type {Boolean|Object}
   * @property {String} username The username to log in with.
   * @property {String} password The password to log in with.
   */
  static get login() {
    return {
      username: 'atscm',
      password: '',
    };
  }
}

```

</details>

**Comment by FKurniawanHuisman** [comment]
Hello, any update regarding this issue?

**Comment by LukasHechenberger** [comment]
Hello, sorry for the delay. 
It seems like we're running into some issues when upgrading to atvise server 3.4.x. The most obvious is issue #330 (XML parsing errors in standard resources) which is why I first focussed on this one. Once I've published a fix at the beginning of next week I'll write some integration tests with atvise server 3.4.0 and start working on this one. Thanks for reporting and your patience.

**Comment by FKurniawanHuisman** [comment]
No problem. Thank you for the feedback, we're looking forward for the fix.

**Comment by LukasHechenberger** [comment]
Finally fixed in *v1.0.0-beta.49* 🎉. Run `atscm update --beta` to update.

Thanks again for reporting @FKurniawanHuisman.

**Comment by FKurniawanHuisman** [comment]
Thank you @LukasHechenberger!
